app/controllers/casework/reassigns_controller.rb

```diff
@@ -35,5 +36,14 @@ def current_assignment
         assignment_id: params[:crime_application_id]
       )
     end
+
+    def require_work_stream_flash
```

Comment:
can we make these generic? if we specified constants in the controllers themselves? Just a thought :)